draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -939,39 +939,29 @@ https://www.example.org
 <section title="Message Body" anchor="message.body">
   <x:anchor-alias value="message-body"/>
 <t>
-   The message body (if any) of an HTTP message is used to carry the
-   payload body of that request or response.  The message body is
-   identical to the payload body unless a transfer coding has been
-   applied, as described in <xref target="header.transfer-encoding"/>.
+   The message body (if any) of an HTTP message is used to carry the payload
+   body (<xref target="payload.body"/>) of that request or response. The
+   message body is identical to the payload body unless a transfer coding has
+   been applied, as described in <xref target="header.transfer-encoding"/>.
 </t>
 <sourcecode type="abnf7230"><iref primary="true" item="Grammar" subitem="message-body"/>
   <x:ref>message-body</x:ref> = *OCTET
 </sourcecode>
 <t>
-   The rules for when a message body is allowed in a message differ for
-   requests and responses.
+   The rules for determining when a message body is present in a HTTP/1.1
+   message differ for requests and responses.
 </t>
 <t>
-   The presence of a message body in a request is signaled by a
+   The presence of a message body in a HTTP/1.1 request is signaled by a
```

Comment:
Not sure we need to say HTTP/1.1 everywhere - this is the topic of this spec, right?

Reply:
I took that one out, but left the first one to remind readers that this is 1.1-specific.